internal/run.bzl

```diff
@@ -90,6 +90,7 @@ postcss_run = rule(
         ),
         "output_name": attr.string(default = ""),
         "additional_outputs": attr.output_list(),
+        "data": attr.label_list(allow_files = True),
```

Comment:
Is `data` not automatically consumed by Bazel for rules?

Reply:
It's not for custom rules, only built-in ones.